ocp_resources/job.py

```diff
@@ -63,3 +84,24 @@ def to_dict(self):
                     self.res["spec"]["template"]["spec"][
                         "restartPolicy"
                     ] = self.restart_policy
+
+    def delete(self, wait=False, timeout=TIMEOUT_4MINUTES, body=None):
+        """
+        Delete Job object
+
+        Args:
+            wait (bool): True to wait for Job to be deleted.
+            timeout (int): Time to wait for resource deletion.
+            body (dict): Content to send to delete().
+
+        Returns:
+            bool: True if delete succeeded, False otherwise.
+        """
+        body = (
```

Comment:
this overwrites `body` which is passed to the function

Reply:
ack